aip/0142.md

```diff
@@ -65,6 +69,26 @@ message is used. For example, a Python-based tool could convert timestamps to
 [datetime][py_datetime] objects and durations to [timedelta][py_timedelta]
 objects.
 
+### Relative time segments
+
+In some cases, it may be necessary to represent a time segment inside a stream.
+In these cases, the [`google.protobuf.Duration`][duration] type **should** be
+used, and the field name **should** end with `_offset`.
+
+To illustrate this, consider a resource representing a segment of an audio
```

Comment:
This sentence seems unnecessary.

Reply:
Err. I was copying what we have above for the example of the flight duration... ("To illustrate the distinction between timestamps and durations, consider a flight record:")

Reply:
Fair, I should probably remove that too.

Reply:
That makes sense. Perhaps we should remove both. Not concerned though.